cookbook/entities/acl.rst

```diff
@@ -0,0 +1,336 @@
+   permissions.
+
+#. If you do not want to protect any controller methods or if you prefer to keep the definition of
+   your ACLs separated from the application code, you can define them using some YAML config in a
+   file named ``acl.yml``:
+
+   .. code-block:: yaml
+       :linenos:
+
+       # src/AppBunde/Resources/config/acl.yml
+       app_task_create:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: CREATE
+
+       app_task_delete:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: DELETE
+
+       app_task_edit:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: EDIT
+
+       app_task_view:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: VIEW
+
+.. sidebar:: Security Actions that Are not Related to an Entity
+
+    You can also create access control lists that are only used to protect certain actions that are
+    not related to an entity. To do that just set the type of the ACL to ``action`` instead of
+    ``entity``:
+
+    .. code-block:: php
+        :linenos:
+
+        // src/AppBunde/Controller/PageController.php
+        namespace AppBunde\Controller;
+
+        use Oro\Bundle\SecurityBundle\Annotation\Acl;
+        use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+        class PageController extends Controller
+        {
+            /**
+             * @Acl(
+             *     id="app_static_pages",
+             *     type="action"
+             * )
+             */
+            public function showAction()
+            {
+                // ...
+            }
+
+            // ...
+        }
+
+    When configuring the ACL using the YAML config format, you also have to set the controller to
+    bind the ACL to using the ``bindings`` option:
+
+    .. code-block:: yaml
+        :linenos:
+
+        # src/AppBunde/Resources/config/acl.yml
+        app_static_pages:
+            type: action
+            bindings:
+                class: AppBunde\Controller\PageController
+                method: showAction
+
+    .. seealso::
+
+        All configuration options are explained in full details in the :doc:`/reference/annotation/acl`,
+        :doc:`/reference/annotation/acl_ancestor`, and :doc:`ACL YAML format </reference/format/acl>`
+        reference.
+
+.. _coobook-entities-acl-check:
+
+Perform Access Checks
+---------------------
+
+Once you have configured the ACLs you can protect all parts of your application. Anywhere in your
+PHP code you can use the ``isGranted()`` method of the ``oro_security.security_facade`` service
+(which is an instance of the :class:`Oro\\Bundle\\SecurityBundle\\SecurityFacade` class):
+
+.. code-block:: php
+    :linenos:
+
+    $securityFacade = $this->container->get('oro_security.security_facade');
+
+    if ($securityFacade->isGranted('app_task_create')) {
```

Comment:
Please add more examples that will describe what you can set different types of data to this method